test/integration/policy-imagemanifest_test.go

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) 2022 Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+
+package integration
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	policiesv1 "github.com/stolostron/governance-policy-propagator/api/v1"
+	"github.com/stolostron/governance-policy-propagator/test/utils"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8stypes "k8s.io/apimachinery/pkg/types"
+
+	"github.com/stolostron/governance-policy-framework/test/common"
+)
+
+const policyIMVURL = "https://raw.githubusercontent.com/stolostron/policy-collection/main/stable/SI-System-and-Information-Integrity/policy-imagemanifestvuln.yaml"
+const policyIMVName = "policy-imagemanifestvuln"
+const subName = "container-security-operator"
+
+var ImageManifestLabel = Label("policy-collection", "stable", "SVT")
```

Comment:
Remove SVT label as it is not necessary. SVT will run all test cases by default.